app/scripts/templates/sign_in.mustache

```diff
@@ -64,7 +64,7 @@
       {{^suggestedAccount}}
         <form novalidate>
         <div class="input-row">
-          <input type="email" class="email{{#email}} prefilled{{/email}}" placeholder="{{#t}}Email{{/t}}" value="{{ email }}" spellcheck="false" {{^email}}autofocus{{/email}}>
+          <input name="email" type="email" class="email{{#email}} prefilled{{/email}}" placeholder="{{#t}}Email{{/t}}" value="{{ email }}" spellcheck="false" {{^email}}autofocus{{/email}}>
```

Comment:
So, should the `value = {{ email }}` be removed here, since the prefill mixin will do it?

And if so, then same with the password input below?